services/storage/src/simcore_service_storage/handlers_files.py

```diff
@@ -192,11 +192,10 @@ async def upload_file(request: web.Request) -> web.Response:
     )
     if query_params.file_size is None and not query_params.is_directory:
         # return v1 response
+        response = {"data": {"link": jsonable_encoder(links.urls[0], by_alias=True)}}
+        log.debug("Returning v1 response: %s", response)
         assert len(links.urls) == 1  # nosec
-        return web.json_response(
-            {"data": {"link": jsonable_encoder(links.urls[0], by_alias=True)}},
-            dumps=json_dumps,
-        )
+        return web.json_response(response, dumps=json_dumps)
```

Comment:
We should get these utils into `servicelib.aiohttp` at some point

https://github.com/itisfoundation/osparc-simcore/blob/2e5d96c42b89f6edd957f47db8c016a198719d56/services/web/server/src/simcore_service_webserver/utils_aiohttp.py#L64-L77